crates/ursa-rpc-service/src/api.rs

```diff
@@ -239,6 +257,28 @@ where
             ))),
         }
     }
+
+    async fn submit_narwhal_txn(&self, txn: TransactionProto) -> Result<()> {
+        let mut client = TransactionsClient::connect(self.mempool_address.clone())
+            .await
+            .unwrap();
+        if let Err(e) = client.submit_transaction(txn).await {
+            Err(anyhow!(format!("Failure sending transacation: {e:?}")))
+        } else {
+            Ok(())
+        }
```

Comment:
The `format!` isn't needed when using `anyhow!`.
```suggestion
        client.submit_transaction(txn).await.map_err(|e| anyhow!("Failure sending transacation: {e:?}"))
```

Reply:
fixed. But kept the if, because we really dont want to return anything besides Ok(()) here